tensorflow/lite/CMakeLists.txt

```diff
@@ -415,6 +415,11 @@ target_link_libraries(tensorflow-lite
     ruy
     ${TFLITE_TARGET_DEPENDENCIES}
 )
+
+if (BUILD_SHARED_LIBS)
```

Comment:
how to set BUILD_SHARED_LIBS ?

Reply:
https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html

It's standard cmake switch